apps/web/.env.development

```diff
@@ -23,3 +23,6 @@ HTTPS_ENABLED=false
 PORT=3000
 # Default : 3443
 HTTPS_PORT=3443
+
+# API Endpoint
+API_ENDPOINT=http://localhost:3000
```

Comment:
`API_ENDPOINT` is not super clear, maybe rename to `API_HOST`?

Reply:
I will update.